doc/user/zebra.rst

```diff
@@ -1241,36 +1241,103 @@ For protocols requiring an IPv6 router-id, the following commands are available:
 
 .. _zebra-sysctl:
 
-Expected sysctl settings
+sysctl settings
```

Comment:
This seems to be fixed as well. I'm about `=======`

Reply:
I'm not sure what you're saying here. Can you clarify if something needs to be done?

Reply:
![Screenshot from 2021-09-06 20-35-23](https://user-images.githubusercontent.com/3352707/132249486-0f6d1d7e-dfc4-4a2d-b1ad-10d46215a5fe.png)

Too much `=`.

Reply:
resolved in latest force push